src/styles/global.css

```diff
@@ -20,6 +20,43 @@ footer {
   }
 }
 
+@media (prefers-color-scheme: dark){
+  
+  header {
```

Comment:
That sounds good though the orange would make it easier for others trying to have a dark theme with a DDev logo. Any chance we could adopt the orange as a dark equivalent to the blue?  I kind of like the orange as an easier alternative and visual indication we are in the dark theme.

Reply:
The entire layout being inverted seems strong enough an indication to me that dark mode is active and I wouldn’t recommend altering core brand colors to suit dark mode—but I’d leave that up to @rfay.

The green external link icons are a unique departure too, but I don’t have any objection there.

Reply:
that's true for ddev.com but on pages where only the logo represents the brand. it could be the dark version of our logo so we don't have the same issue some of the other logos had.

Reply:
I think we’re describing the same thing! (And most of those sponsor logos could have alternate versions in the same way—need to create the variations but I could either add them here or in a separate PR.)

This is what I’m trying to describe doing with the DDEV logo that doesn’t introduce new brand colors:

<img width="406" alt="ddev-logo-reversed-example" src="https://github.com/ddev/ddev.com/assets/2488775/9d33fb7a-1c69-4bac-8316-fde57556dbdb">

This is an optimized SVG for exactly that (no background) if you want to use it:

![ddev-logo-reversed](https://github.com/ddev/ddev.com/assets/2488775/2f3894d6-0afd-4210-b17f-661c9ce17788)

Could swap the whole thing or be more efficient and select only the text nodes within the SVG.